tests/testthat/test-legend.R

```diff
@@ -0,0 +1,52 @@
+context("legend")
+
+test_that("easy_remove_legend works with no argument", {
+  easy_res <- ggplot(mtcars, aes(wt, mpg, colour = cyl, size = hp)) +
+    geom_point() + easy_remove_legend()
+  
+  hard_res <- ggplot(mtcars, aes(wt, mpg, colour = cyl, size = hp)) +
+    geom_point() + theme(legend.position = "none")
+  
+  expect_equal(easy_res, hard_res)
```

Comment:
Should we `expect_identical()`? I haven't tested whether or not that's actually true yet.

Reply:
Yeah I haven't tried expect_identical but that may be appropriate here!